src/workloads/c2c/CollectionCopy.yml

```diff
@@ -0,0 +1,75 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This workload is a short version load used to test Mongosync Collection Copy stage performance.
+  The workload starts Mongosync on a cluster with an preloaded initial dataset.
+
+Keywords:
+- c2c
+- replication
+- collection copy
+- cluster to cluster sync
+
+# These two values should match those are the top of MixPhases.yml
+dbname: &dbname mongosync_test
+DocumentCount: &NumDocs 100000
```

Comment:
Is this enough data to get a stable number for collection copy speed?

Reply:
Yea, I agree 100K docs might not be enough. I am changing it to 1M.

Reply:
Sounds good! Was there any difference in the cea or collection copy test results with 1M docs?

Reply:
Yea, Collection copy seems 1000 times faster than CEA... I expected it would be faster, but wasn't aware it's that fast.

Reply:
Oh I see - nice finds!

When comparing the 100k docs to the 1M docs version of the test, is there any noticeable difference in results or runtime of the test? I'm just wondering if it worth extra runtime for additional docs, or if 100k is enough.

Reply:
I saw there is no big difference between these two datasets regarding the metrics and test execution time. So I think we can keep the 1M data set here to be more confident.

Reply:
Sounds good!